sushy_oem_idrac/resources/manager/manager.py

```diff
@@ -79,9 +82,16 @@ class DellManagerExtension(oem_base.OEMResourceBase):
     RETRY_COUNT = 10
     RETRY_DELAY = 15
 
-    @property
-    def import_system_configuration_uri(self):
-        return self._actions.import_system_configuration.target_uri
+    def _get_import_system_configuration_element(self):
```

Comment:
Do we need to make this property a callable method? May be we could raise exception from within the property?